test/functional/test_short_deck.py

```diff
@@ -82,3 +85,53 @@ def test_short_deck_2():
     # Only one player left, so game state should be terminal.
     assert state.is_terminal, "state was not terminal"
     assert state._betting_stage == "terminal"
+
+
+@pytest.mark.parametrize(
+    "n_players",
+    [
+        pytest.param(0, marks=pytest.mark.xfail),
+        pytest.param(1, marks=pytest.mark.xfail),
+        2,
+        3,
+        4,
+    ],
+)
+def test_short_deck_3(n_players: int):
+    """Check the state fails when the wrong number of players are provided.
+
+    Test the short deck poker game state works as expected - make sure the
+    order of the players is correct - for the pre-flop it should be
+    [-1, -2, 0, 1, ..., -3].
+    """
+    from pluribus.games.short_deck.player import ShortDeckPokerPlayer
+    from pluribus.games.short_deck.state import ShortDeckPokerState
+    from pluribus.poker.pot import Pot
+
+    pot = Pot()
+    players = [
+        ShortDeckPokerPlayer(player_i=player_i, pot=pot, initial_chips=10000)
+        for player_i in range(n_players)
+    ]
+    state = ShortDeckPokerState(players=players, load_pickle_files=False)
+    order = list(range(n_players))
+    player_i_order = {
+        "pre_flop": order[-2:] + order[:-2],
```

Comment:
Same thing here as previously mentioned about the order perhaps? `order[2:]+order[:2]`